### PR TITLE
Fix wrong sql queries

### PR DESCRIPTION
**Pull request by Jesus89**
From https://github.com/CartoDB/cartoframes/pull/1675

Extra: improve unit tests speed

### CODE REVIEW

File: cartoframes/io/managers/context_manager.py [hunk 1]
```diff
@@ -394,18 +396,18 @@ def _not_reserved(column):
...
+        columns=','.join(columns))
 
 
 def _create_table_from_query_query(table_name, query):
     return 'CREATE TABLE {table_name} AS ({query})'.format(table_name=table_name, query=query)
 
 
 def _cartodbfy_query(table_name, schema):
-    return "SELECT CDB_CartodbfyTable('{schema}', '{table_name}')".format(
+    return 'SELECT CDB_CartodbfyTable(\'{schema}\', \'{table_name}\')'.format(
```
Comment (antoniocarlon) [inline cartoframes/io/managers/context_manager.py:410]: Is this change needed? The query was more legible before

**Review by antoniocarlon — APPROVED** [review]
One minor comment

File: cartoframes/io/managers/context_manager.py (hunk 1 above)
Comment (Jesus89) [reply cartoframes/io/managers/context_manager.py:410]: I prefer using ' ' and then skip to be consistent with other parts of the code
